rfcs/react-components/components/simple-component-implementation.md

```diff
@@ -0,0 +1,124 @@
+
+There's one key aspect that Persona adds that makes creating the component compelling, unifying the look and feel across apps. To achieve this, we don't need the image and icon media options, we only need the Avatar and PresenceBadge (it also makes more sense that a persona only showcases person-related information). This then is the reason why the already implemented Persona only supports these slots. But there's still the question about how to add the look and feel of Fluent to media objects that have an image or icon.
+
+This is where this RFC will decide how to proceed with this portion of the problem.
+
+## Problem statement
+
+As we start working on new components, there might be some components that are simple and could be built with a simple layout and the already finished components. When creating these components, we currently don't have a guidance on whether to let the user build the component or us provide the component. This RFC offers three options, one that will require the user to build the components using instructions (recipes), another where we will provide the implementation of the component, and a combination of both option 1 and 2 providing a partial implementation and a recipe.
+
+#### What makes a component a candidate for this RFC's solution?
+
+- Its implementation is simple.
+- The desired design is achievable adjusting the props of the inner components.
+- It doesn't have complex accessibility requirements.
+- It doesn't have strict design requirements.
+
+## Detailed Design or Proposal
+
+### Option 1: Recipes section in docsite
+
+In this option we will add a section to our docs detailing how to create the component with our already created components.
+
+A great example of a recipe is the [Media Objects recipe](https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Media_objects) by MDN.
+
+#### Folder structure inside docs
+
+```
+\- Concepts
+  |- Introduction
+  |- Developer
+  |- Migration
+  |- Recipes
+    |- Recipe #1
+    |- Recipe #2
+\- Theme
+  |- Color
+  |- Motion and Size
+  |- Typography
+\- Components
+  |- ...
+\- Preview Components
+  |- ...
+```
+
+#### Pros
+
+- No need for a new package in our repo.
+- In the case of a component like persona where its design requires us to change the icon's size, we would provide best practices section that could include a size table for reference.
+
+#### Cons
+
+- Allows for issues where the user-created component doesn't look like the design.
+- Could increase a11y issues in user-created components.
+- Could be overwhelming for the user to follow the recipe.
+
+### Option 2: We provide the component
+
+For this option, we would implement the component and provide it like any other component.
```

Comment:
just to clarify, do you mean that we would implement `Persona` or a generic `Media` layout component ?

Reply:
If we go with this option, we will provide a Media Layout component and a Persona component.

Reply:
Have we checked with design if there is an appetite for a reusable media layout on their side? It would requirement commitmment from both us (to implement and maintain it) and design (to design with it in mind)